ppml/docs/examples.md

```diff
@@ -293,6 +293,99 @@ bash bigdl-ppml-submit.sh \
 ```
 </details>
 
+<details><summary>Trusted SimpleQuery With Multiple KMS/Datasource</summary>
+
+If you have multiple data sources that use different keys, you can also use the `initPPMLContextMultiKMS` method to initialize PPML Context with support for multiple Key Management Systems and data sources.   
+
+You just need to submit the configurations for the KMS and data sources in a manner similar to the following example.
+
+For ***KMS***, you should first submit the number of kms `spark.bigdl.kms.multikms.instance`, then submit parameters for every KMS.
+ - Firstly, submit `spark.bigdl.kms.multikms.instance` \
+ **spark.bigdl.kms.multikms.instance**  num of your kms
+
+ - Then, submit configurations for each individual kms, `type` and `name` should be specified for every kms. \
+ **{i}** means it is a configuration for ith KMS \
+ **spark.bigdl.kms.multikms.type{i}:**  KMS type of this KMS \
+ **spark.bigdl.kms.multikms.name{i}:**  name of this KMS
+
+There are also their own unique parameters to be specified for different kinds of KMS. 
+
+ - For a simple KMS \
+ **spark.bigdl.kms.multikms.simple.id{i}:**  simple KMS APPIP  \
+ **spark.bigdl.kms.multikms.simple.key{i}:**  simple KMS APIKEY
+
+ - for a EHSM KMS \
+ **spark.bigdl.kms.multikms.ehs.ip{i}:**  ehsm ip \
+ **spark.bigdl.kms.multikms.ehs.port{i}:**  ehsm port \
+ **spark.bigdl.kms.multikms.ehs.id{i}:**  ehsm APPID \
+ **spark.bigdl.kms.multikms.ehs.key{i}:**  ehsm APIKEY 
+
+ - for a Azure KMS \
+ **spark.bigdl.kms.multikms.azure.vault{i}:** azure KMS KeyVault \
+ **spark.bigdl.kms.multikms.azure.clientId{i}:** azure KMS clientId
+
+
+For ***data sources***, you should first submit the number of data sources `spark.bigdl.kms.datasource.instance`
+
+ - Firstly, submit the number of data sources \
+ **spark.bigdl.kms.datasource.instance** = num of your data sources
+
+ - Then submit the configurations for each data source in turn.\
+ **{i}** means it is a configuration for ith data source \
+ **spark.bigdl.kms.datasource{i}.name:** name of this data source \
+ **spark.bigdl.kms.datasource{i}.kms:**  KMS to be used. Should match a KMS name registered previously  \
+ **spark.bigdl.kms.datasource{i}.inputpath:** input path of this data source \
+ **spark.bigdl.kms.datasource{i}.outputpath:** output path of this data source \
+ **spark.bigdl.kms.datasource{i}.primary:** primary key path of this data source \
+ **spark.bigdl.kms.datasource{i}.data:** data key path of this data source 
+
+
+local mode
+<p align="left">
+  <img src="https://user-images.githubusercontent.com/61072813/174703141-63209559-05e1-4c4d-b096-6b862a9bed8a.png" alt="data lifecycle" width='250px' />
```

Comment:
need a new architecture chart to illustrate the multi KMS scenario, where there are both ehsmKMS and simpleKMS